cocotb/clock.py

```diff
@@ -50,7 +50,7 @@ def log(self):
 class Clock(BaseClock):
     """Simple 50:50 duty cycle clock driver.
 
-    Instances of this class should call its :meth:`start` method and fork the
+    Instances of this class should call its :meth:`start` method and :func:`fork` the
```

Comment:
This doesn't actually produce a link, but that can be fixed in a follow up. Perhaps we have no documentation for `fork`?

Reply:
Yes, ``fork`` is missing a docstring, I opened #1176 for that. Could you suggest a docstring there? I'm not sure I'd get the wording nice.